tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/homescreen_settings.py

```diff
@@ -10,13 +10,18 @@
 class HomescreenSettings(Base):
 
     _page_locator = (By.ID, 'homescreens')
+    _change_icon_layout_locator = (By.CSS_SELECTOR, 'select[name="grid.layout.cols"]')
+    _change_icon_choices_locator = (By.CSS_SELECTOR, '.value-selector-options-container > [role="option"]')
+    _confirm_layout_button_locator = (By.CSS_SELECTOR, 'button[data-l10n-id="ok"]')
+
     _change_homescreen_page_locator = (By.ID, 'homescreens-list')
     _change_homescreen_locator = (By.CSS_SELECTOR, '[data-l10n-id="change-home-screen"]')
     _get_more_homescreen_locator = (By.CSS_SELECTOR, '[data-l10n-id="get-more-home-screens"]')
+    _pick_homescreen_cancel_button_locator = (By.CSS_SELECTOR, '#action-menu-list button[data-l10n-id="cancel"]')
 
     _wallpaper_preview_locator = (By.CSS_SELECTOR, '.wallpaper-preview')
     _wallpaper_pick_locator = (By.CLASS_NAME, 'wallpaper-button')
-    _pick_cancel_button_locator = (By.CSS_SELECTOR, '#action-menu-list button[data-l10n-id="cancel"]')
+    _wallpaper_pick_cancel_btn_locator = (By.CSS_SELECTOR, '#action-menu-list button[data-l10n-id="cancel"]')
```

Comment:
Since this has an id=action-menu-list button, that should be enough, so you can use the ID selector here.